scripts/sanitycheck

```diff
@@ -1224,23 +1225,29 @@ class MakeGenerator:
         handler = None
         if type == "qemu":
             handler = QEMUHandler(instance)
+            handler.type = "qemu"
```

Comment:
Make the .type attribute mandatory by adding it to the constructors instead:
```
handler = QEMUHandler(instance, type_str)
```
It could be used for a future `__str__()`

(BTW "type" is a builtin, so not the best name)

Reply:
+1